include/dlaf/matrix/tile.h

```diff
@@ -213,7 +213,7 @@ Tile<const T, device>::Tile(Tile&& rhs) noexcept
 template <class T, Device device>
 Tile<const T, device>::~Tile() {
   if (p_) {
-    if (std::uncaught_exception())
+    if (std::uncaught_exceptions() > 0)
```

Comment:
@rasolca I didn't revert this change yet. Should I? It may need some pragmas around it to disable the warning.